joomla/joomla-cms
# [4.0] Translate stages via language files

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/21476#issuecomment-411694584.

### Summary of Changes
Now one can use languages constants to have multilingual states



### Testing Instructions
Create a state called "JPUBLISHED" and check it everywhere in the system (state list, article edit form, article list, transition edit form, ...)


### Expected result
The language constant will be translated


### Actual result
No translation at all
